pkg/skaffold/runner/runcontext/v2/context.go

```diff
@@ -212,10 +213,14 @@ func GetRunContext(opts config.SkaffoldOptions, configs []*latestV2.SkaffoldConf
 	kubeContext := kubeConfig.CurrentContext
 	logrus.Infof("Using kubectl context: %s", kubeContext)
 
-	// TODO(dgageot): this should be the folder containing skaffold.yaml. Should also be moved elsewhere.
-	cwd, err := os.Getwd()
-	if err != nil {
-		return nil, fmt.Errorf("finding current directory: %w", err)
+	var cwd string
```

Comment:
sorry, reset vote due to https://github.com/GoogleContainerTools/skaffold/pull/6245#pullrequestreview-710707523.

Reply:
Thanks. Reverted the workdir/skaffold.yaml changes.